.changeset/loud-dolls-appear.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/shared-object-base": minor
+"__section": legacy
+---
+Added an optional boolean parameter `fullTree` to `SharedObject`'s `summarizeCore` method
```

Comment:
Our most recent guidance for changeset titles is to not use `inlineCode` syntax (to prevent the generated changelogs from appearing too noisy). This still needs to be documented in a more accessible place, but here are the guidelines we're using when reviewing: https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/on-call/release/release-prep

```suggestion
Added an optional boolean parameter "fullTree" to SharedObject's summarizeCore method
```